Documentation/user-guides/webhook.md

```diff
@@ -72,5 +77,71 @@ webhooks:
     sideEffects: None
 ```
 
-The `caBundle` contains the base64-encoded CA certificate used to sign the
-webhook's certificate.
+### `/admission-prometheusrules/mutate`
+
+The endpoint `/admission-prometheusrules/mutate` ensures that integers and boolean yaml data elements are cooerced into strings.
+
+The following example deploys the mutating admission webhook:
+```yaml
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: prometheus-operator-rulesmutation
+webhooks:
+  - clientConfig:
+      caBundle: SOMECABASE64ENCODED==
+      service:
+        name: prometheus-operator
```

Comment:
I wonder should we update these to the HA setup and mention that here? That would be `prometheus-operator-admission-webhook`

@simonpasquier - thoughts?

Reply:
I would advise that this PR documents the missing endpoints and a follow-up PR would document the HA setup with `prometheus-admission-webhook`. WDYT?

Reply:
Yeah.  I want to keep it tight.

Reply:
sounds good